src/org/etools/j1939_84/modules/ReportFileModule.java

```diff
@@ -81,28 +79,6 @@ public void addOutcome(int partNumber, int stepNumber, Outcome outcome, String m
         onResult(new ActionOutcome(outcome, message).toString());
     }
 
-    @Override
-    public void beginPart(PartResult partResult) {
-        summaryModule.beginPart(partResult);
-    }
-
-    @Override
-    public void beginStep(StepResult stepResult) {
-
-    }
-
-    @Override
-    public void endPart(PartResult partResult) {
-        // onResult(NL);
-        // onResult("End Part " + partResult);
-    }
-
-    @Override
-    public void endStep(StepResult stepResult) {
-        // onResult(NL);
-        summaryModule.endStep(stepResult);
```

Comment:
I think this needs to remain

Reply:
Moved it to the stepController as suggested below.